components/calendar/Header.tsx

```diff
@@ -178,8 +180,8 @@ export default class Header extends React.Component<HeaderProps, any> {
     );
   };
 
-  headerRenderCustom = (): React.ReactNode => {
-    const { headerRender, type, onValueChange, value } = this.props;
```

Comment:
I think this change is unnecessary.

Reply:
I change here because
`
const { headerRender, type, onValueChange, value } = this.props;
`
Type of headerRender is    HeaderRender **| undefined**

If I pass headerRender as function argument, because the func is called by
`
headerRender ? this.headerRenderCustom(headerRender) : (...)
`
the type of headerRender is HeaderRender

I just want to avoid the repeated undefined check in TS